### PR TITLE
feat: add command for generating README.md

### LINKED ISSUE #2818: Link out to each github repo hosting the specific client lib
This repo when mentioning "Spanner Nodejs" should have a link to where the code lives as well as a link back to the google cloud webpage.

### PR DESCRIPTION
**Pull request by bcoe**
Add tool for generating README so that it's easier to keep up-to-date, and so we can use this as the landing page from cloud.google.com when linking to all Node.js libraries.

fixes #2818

### CODE REVIEW

File: bin/generate-readme.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+// grab files using GitHub API.
+async function getContent (owner, repo, path) {
+  const headers = {}
+  if (argv.token) headers.authorization = `token ${argv.token}`
+  const resp = await fetch(`${apiUrl}/repos/${owner}/${repo}/contents/${path}`, {
+    headers: headers
+  })
+    .then(res => {
```
Comment (JustinBeckwith) [inline bin/generate-readme.js]: Mixing promises and async functions feels weird.  Can we just use `await` here?

File: bin/generate-readme.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      if (res.status !== 200) {
```
Comment (JustinBeckwith) [inline bin/generate-readme.js]: If you used `gaxios` it would just work this way 😛

File: bin/generate-readme.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        const err = Error(`unexpected status = ${res.status}`)
+        err.status = res.status
+        throw err
+      }
+      return res.json()
+    })
+  return JSON.parse(Buffer.from(resp.content, 'base64').toString('utf8'))
```
Comment (JustinBeckwith) [inline bin/generate-readme.js]: Can we use 2 lines here to make it more readable?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-cloud-node",
+  "version": "1.0.0",
```
Comment (JustinBeckwith) [inline package.json]: Could we lose the `version`, and add a `private: true` in here?

**Comment by bcoe** [comment]
doesn't automate things yet, but at least makes generating a new README.md a single command.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "Google Cloud client libraries for Node.js",
+  "main": "bin/generate-readme.js",
```
Comment (JustinBeckwith) [inline package.json]: This is going to confuse people.  I don't expect anyone to install this.  I'd much rather just have an `npm run` command

File: package.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  "bin": "bin/generate-readme.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "fix": "standard --fix",
```
Comment (JustinBeckwith) [inline package.json]: Eugh. I see the appeal here, but it is inconsistent with literally every other module we have.  I'll let it pass if you use `semistandard` instead.

File: bin/generate-readme.js (hunk 2 above)
Comment (bcoe) [reply bin/generate-readme.js]: wasn't sure when to reach for teeny-request or gaxios ... so reached for `node-fetch` :stuck_out_tongue:

**Pull request merged**